engine/src/vsfilesystem.cpp

```diff
@@ -707,14 +725,22 @@ void LoadConfig( string subdir )
         BOOST_LOG_TRIVIAL(info) << boost::format("DATADIR - Found a datadir in config, using : %1%") % game_options.datadir;
         datadir = game_options.datadir;
     } else {
-        BOOST_LOG_TRIVIAL(info) << boost::format("DATADIR - No datadir specified in config file, using : %1%") % datadir;
+        if (true == legacy_mode) {
+            BOOST_LOG_TRIVIAL(info) << boost::format("DATADIR - No datadir specified in config file, using : %1%") % datadir;
+        } else {
+            BOOST_LOG_TRIVIAL(fatal) << boost::format("DATADIR - No datadir specified in config file");
+        }
     }
 
     string universe_file = datadir + "/" \
         + vs_config->getVariable( "data", "universe_path", "universe" ) + "/" \
         + vs_config->getVariable( "general", "galaxy", "milky_way.xml" );
     BOOST_LOG_TRIVIAL(debug) << "Force galaxy to " << universe_file;
-    Galaxy galaxy = Galaxy(universe_file);
+    try {
+        Galaxy galaxy = Galaxy(universe_file);
+    } catch (std::exception &e) {
+        BOOST_LOG_TRIVIAL(fatal) << boost::format("Error while loading configuration. Did you specifcy the asset directory? Error: %1%") % e.what();
```

Comment:
Three things:

1. Maybe catch a more specific exception type?
2. Call `VSExit`, as above.
3. `specifcy` should be `specify`.

Reply:
okay...so here's the error I'm getting:
```
GOT SUBDIR ARG = 
Found MODDIR = /mods
USING HOMEDIR : /home/bmeyer/.vegastrike As the home directory 
CONFIGFILE - Found a config file in home directory, using : /home/bmeyer/.vegastrike/vegastrike.config
Force galaxy to /universe/milky_way.xml
terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::property_tree::xml_parser::xml_parser_error> >'
  what():  /universe/milky_way.xml: cannot open file
Aborted (core dumped)
```
however, I'm not sure that's an appropriate error to catch as it likely won't be the_only_ error that could possibly happen. That said, I had to remove a bunch of stuff to get the error to happen again; still catching the errors here would be a good idea.

Reply:
yeah, maybe just catching exceptions in general is OK here